test/api-digester/stability-stdlib-abi-with-asserts.test

```diff
@@ -46,3 +46,8 @@ Struct _ObjectRuntimeFunctionCountersState is a new API without @available attri
 Struct _RuntimeFunctionCounters is a new API without @available attribute
 Func _swift_isImmutableCOWBuffer(_:) is a new API without @available attribute
 Func _swift_setImmutableCOWBuffer(_:_:) is a new API without @available attribute
+Func BinaryInteger.isPower(of:) is a new API without @available attribute
+Func BinaryInteger._slowIsPower(of:) is a new API without @available attribute
+Func BinaryInteger._isPowerOf(powerOfTwo:) is a new API without @available attribute
+Var BinaryInteger._isPowerOfTen is a new API without @available attribute
+Var BinaryInteger._isPowerOfTwo is a new API without @available attribute
```

Comment:
These should normally be added to the "…-without-asserts.test" file instead, so that they also apply when building with the `--no-assertions` flag. However, your new APIs do need either `@_alwaysEmitIntoClient` for back-deployment, or `@available` with version 9999 placeholders.

Reply:
Thanks a lot for your information, Ben! I should have read `docs/StandardLibraryProgrammersManual.md` earlier, where the 9999 `@available` placeholder is presented. Now the patch is updated to revert changed in api-digest, and use non-ABI-breaking attributes to those new APIs.

Reply:
I think the public `isPower(of:)` function can also use `@_alwaysEmitIntoClient` if you want. Then the `#available` guards aren't needed.

Reply:
Done~